great-idea/contact.html

```diff
@@ -0,0 +1,97 @@
+                        <a href="#">Features</a>
+                        <a href="#">About</a>
+                        <a class="current-page" href="#">Contact</a>
+                    </nav>
+                    <a class="home-page" href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+                </header>
+                <div class="header-img-contact"><img class="contact-header-img" src="img/contact-banner.jpg" alt="Our contact header image"></div>
+            </div>
+            <div class="content-contact">
+                <div class="contact-left">
+                    <div class="contact-header">
+                        <p class="contact-header-left">01 /</p>
+                        <p class="contact-header-right">GET IN<br />TOUCH</p>
+                    </div>
+                    <p class="contact-us">We would love to speak to you. Feel free to call, send us an email, Tweet us or simply complete the enquiry form.</p>
+                    <div class="address">123 Way 456 Street<br />Somewhere, USA</div>
+                    <div class="phone">
+                        <img src="/great-idea/img/phone-not-being-used.png" />
+                        <p>(888) 888-8888</p>
+                    </div>
+                    <div class="email">
+                        <img src="/great-idea/img/icons8-envelope-100.png" />
+                        <p>sales@greatidea.io</p>
+                    </div>
+                    <div class="twitter">
+                        <img src="/great-idea/img/twitter.png" />
+                        <p>@GREATIDEA</p>
+                    </div>
+                    <div class="facebook">
+                        <img src="/great-idea/img/facebook.png" />
+                        <p>/GREATIDEA</p>
+                    </div>
+                    <div class="instagram">
+                        <img src="/great-idea/img/instagram-new.png" />
+                        <p>/GREATIDEA</p>
+                    </div>
+                </div>
+                <div class="contact-right">
+                        <div class="contact-header">
+                            <p class="contact-header-left">02 /</p>
+                            <p class="contact-header-right">SEND US<br />A MESSAGE</p>
+                        </div>
+                        <p>Your name <span class="required">(required)</span></p>
+                        <input type="text" name="name" size="50" maxlength="50" />
+
+                        <p>Email address <span class="required">(required)</span></p>
+                        <input type="text" name="email" size="50" maxlength="50" />
+
+                        <p>Telephone number</p>
+                        <input type="text" name="telephone-number" size="50" maxlength="20" />
+
+                        <p>Your message <span class="required">(required)</span></p>
+                        <textarea name="message" cols="50" rows="10"></textarea>
```

Comment:
I love the use of text area here, you don't see used very often but it gives you some awesome properties that are perfect for comment boxes.

Reply:
Yes, some elements are so cool but rarely used.